geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-orc/src/main/scala/org/locationtech/geomesa/fs/storage/orc/utils/OrcAttributeReader.scala

```diff
@@ -217,7 +220,7 @@ object OrcAttributeReader {
           coordinates(i) = new Coordinate(x.vector(offset + i), y.vector(offset + i))
           i += 1
         }
-        sf.setAttribute(attribute, gf.createMultiPoint(coordinates))
+        sf.setAttribute(attribute, gf.createMultiPointFromCoords(coordinates))
```

Comment:
*Blinks* where did this change come from?  (Just curious)

Reply:
the other method is deprecated now, not exactly sure why... since we're pretty tightly tied to geotools versions at this point i figured it was ok to use the newer method.